### ISSUE TITLE
Develop 404.html to Be Language-Oriented Redirection By Default

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#55]
# Description
```
Please provide a short description of what feature you're looking forward to
enhance below. Please include the story behind your idea as well to give a
better visualization of your idea.
```
At the moment, `404.html` is not developed at all. Since ZORALab's Hestia is now capable of self-sustain on Hugo, we should proceed to develop one and always set to languge-oriented redirection by default.

# Expected Behavior
```
Please specify the expected behavior of your requested enhancement. Some great
and helpful pointers are your expected interface (e.g. command patterns,
simple sketches of the user interface, etc).
```
`404.html` is working by default.

# Current Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
`404.html` is not working by default.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 00300e197fda45216570efc0f445da85eacc5908

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
